yasa/hypno.py

```diff
@@ -215,7 +219,7 @@ class Hypnogram:
      '%REM': 8.9713}
     """
 
-    def __init__(self, values, n_stages=5, *, freq="30s", start=None, scorer=None):
+    def __init__(self, values, n_stages=5, *, freq="30s", start=None, scorer=None, proba=None):
         assert isinstance(
```

Comment:
Right after checking that `values` is a list, I would check they are all strings, with a useful error message telling the use that "yasa expects strings for each epoch now". The first thing I tried when playing around was:

```python
>>> yasa.Hypnogram([1, 2, 3, 4])
# AttributeError: 'int' object has no attribute 'upper'
```

I'm quite sure this will happen for many users when upgrading to 0.7, and there should probably be as much hand-holding as possible.

Reply:
Good idea — added a new check